src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs

```diff
@@ -134,6 +134,7 @@ public CollectionView() : base()
         {
             FocusGroup = true;
             SetKeyboardNavigationSupport(true);
+            SetAccessibilityConstructor(Role.List, AccessibilityInterface.None);
```

Comment:
I think you don't have to add the second argument when it's `None` value. 
But, someone else may approve it. :)

Reply:
I agree with Ms. Kim, it is better to remove the second parameter :).

Reply:
However please consider if the CollectionView should not support Selection interface.

Reply:
It support items selection not it's own selection.
Count can be readable and selected item count also readable then should it be value?

Reply:
Note: AccessibilityInterface.Selection PR: https://github.com/Samsung/TizenFX/pull/3110

Reply:
okay I'll wait till it merged :)